with-vue-supabase/README.md

```diff
@@ -0,0 +1,44 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
```

Comment:
To approve this PR the README should follow the format of the file TEMPLATE.md we have in this repository. All integrations have the same README format...